examples/example-vite-svelte-recs/src/dojo/typescript/models.gen.ts

```
@@ -71,40 +71,40 @@
     y: Number;
-}
+};
 export const Vec2Definition = {
     x: RecsType.Number,
     y: RecsType.Number,
 };
 
 // Type definition for `dojo_starter::models::Position` struct
-export interface Position {
+export type Position = {
     player: BigInt;
```

Biome (1.9.4)

[error] Don't use 'BigInt' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'bigint' instead


(lint/complexity/noBannedTypes)